drivers/bluetooth/hci/spi.c

```diff
@@ -194,12 +217,65 @@ static void bt_spi_release_cs(void)
 {
 	gpio_pin_write(cs_dev, GPIO_CS_PIN, 1);
 }
+
+static bool bt_spi_irq_pin_high(void)
```

Comment:
You don't need prefixes (`bt_spi_` for static functions), can you remove that?

Reply:
@carlescufi are you sure because the other functions like bt_spi_configure_cs, bt_spi_kick_cs, bt_spi_release_cs have the bt_spi_ prefix ? and are not introduce by me. If I change the name of my new function, do I need to change this function name also ?

Reply:
@carlescufi I have pushed the new version of patch with this modification. Thanks